frontend/src/components/FormGroupSettings.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {
+  FormGroup,
+  Text,
+  HelperText,
+  HelperTextItem,
+  Alert,
+  AlertActionCloseButton,
+  Hint,
+  HintBody,
+} from '@patternfly/react-core';
+import { MultiSelection } from './MultiSelection';
+import { GroupsConfigField, MenuItemStatus } from 'pages/groupSettings/groupTypes';
+
+type FormGroupSettingsProps = {
+  title: string;
+  body: string;
+  footer: string;
+  fieldId: string;
+  groupsField: GroupsConfigField;
+  items: MenuItemStatus[];
+  handleMenuItemSelection: (newState: MenuItemStatus[], field: GroupsConfigField) => void;
+  handleClose: () => void;
+  error?: string;
+  hintText?: string;
+};
+
+export const FormGroupSettings: React.FC<FormGroupSettingsProps> = ({
+  title,
+  body,
+  footer,
+  fieldId,
+  groupsField,
+  items,
+  handleMenuItemSelection,
+  handleClose,
+  error,
+  hintText,
+}) => {
+  return (
+    <FormGroup fieldId={fieldId} label={title}>
```

Comment:
I noticed this problem in my tolerations PR -- we bastardized the PF types and we are suffering because of it. This is not a good wrapper component. But it is what it is now. Just wanted to let you know.

Reply:
#537